requirements/edx/base.txt

```diff
@@ -49,7 +49,7 @@ cryptography==2.9.2       # via -r requirements/edx/../edx-sandbox/shared.txt, d
 cssutils==1.0.2           # via pynliner
 ddt==1.3.1                # via -c requirements/edx/../constraints.txt, xblock-drag-and-drop-v2, xblock-poll
 decorator==4.4.2          # via pycontracts
-defusedxml==0.7.0rc1      # via -r requirements/edx/base.in, djangorestframework-xml, ora2, python3-openid, python3-saml, safe-lxml, social-auth-core
+defusedxml==0.6.0         # via -r requirements/edx/base.in, djangorestframework-xml, ora2, python3-openid, python3-saml, safe-lxml, social-auth-core
```

Comment:
As far as I can tell, this version regression would only affect Python 3.9: https://github.com/tiran/defusedxml/compare/v0.6.0..v0.7.0.rc1